website/content/tutorials/1-getting-started/_index.md

```diff
@@ -0,0 +1,954 @@
+
+Our Metaschema-enabled tools can parse and generate the different data formats. We specify flags on the `computer` assembly, but all else we define in the `model` of the `assembly`. And within that model, we can define the motherboard `assembly` inline with its own `flag` and `model`. XML has different syntax and semantics from JSON and YAML that would lead to structural limitations we would need to work around ourselves by manually designing separate data formats without Metaschema. With Metaschema, we can use `group-as` declarations  in our definition, and Metaschema-enabled tools will generate and parse instances for us as instances of the same, despite these limitations, in different data formats automatically.
+
+We define the data types for different Metaschema fields and flags. Our Metaschema-enabled tools can leverage pre-compiled schemas or generate their own to enforce `field` and `flag` values that are valid for their type. For example, our Metaschema-enabled tools should accept a valid URI for the `website` field of the vendor `assembly`, but not any arbitrary string. For `byte-size`, they should only accept positive integer values greater than 0, not a decimal point number or string. Metaschema facilitates consistent enforcement of data typing so we developers do not have to.
+
+We also define the minimum and maximum number of elements for the different assemblies, flags, and field with `min-occurs` and `max-occurs` declarations. In our example, we have an opptional `expansion-card` field in the motherboard `assembly`. Our Metaschema-enabled tools will parse or generate instances as valid with optional fields missing. On the other hand, a motherboard `assembly` missing the CPU `field` should throw errors, as should parsing or generating instances with one that one CPU `field` in the JSON, XML, or YAML format.
+
+## Refactoring Metaschema Definitions and Deduplicating Code
+
+We now have a robust information model for a computer we can express in JSON, XML, and YAML data models. But what if we want to enhance the information model? Can we add more information but also refactor to be more expressive while reducing redundancy? With Metaschema, yes we can.
+
+Our stakeholders determine supply chain information is very important. We need to know vendor information for all the different parts of the computer, specifically a company name and where the company is headquartered. This information should be maintained for not just the computer, but all parts and sub-parts. How can we add this to the Metaschema definition?
+
+To get started, we can copy-paste vendor `assembly` into all relevent assemblies, not just the top-level computer assembly.
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<METASCHEMA xmlns="http://csrc.nist.gov/ns/oscal/metaschema/1.0">
```

Comment:
Do we really want to model a metaschema with three or four levels of nesting definitions? I know, after this we show how to factor them out ... maybe a hint to the nervous?

"To get started" on line 416 could say "For now", perhaps providing this hint to at least the keen-eyed among nervous readers.

Reply:
I'll consider this for now bit, but I will have to edit locally since it is not in the diff here for some reason. I think 3 or 4 layers deep is common in our real-world schemas so, for now, we should expect them to get acquainted.  ;-)

Reply:
Also tweaked the for now in https://github.com/usnistgov/metaschema/pull/275/commits/dbbd1cab80eb8d3cd15f9751a488bdf8994325fa.